components/TextFields/src/MDCMultilineTextField.m

```diff
@@ -685,22 +707,17 @@ - (void)setTextInsetsMode:(MDCTextInputTextInsetsMode)textInsetsMode {
 }
 
 - (UITextView *)textView {
-  if (!_textView) {
-    _textView = [[UITextView alloc] initWithFrame:CGRectZero];
-    [self setupTextView];
-    [self setupUnderlineConstraints];
-  }
+  [self textViewSafetyCheck];
   return _textView;
 }
 
 - (void)setTextView:(UITextView *)textView {
```

Comment:
Add a comment explaining the magic here.  Perhaps something along the lines of:
We silently replace the textView with an MDCIntrinsicHeightTextView version because x, y and z.

I can see people getting confused when they assume that the textView they are setting here end up NOT being the textView in the component. :/

Reply:
Done.